### PR TITLE
add Xiaomi Redmi S2 (ysl) manifest

### PR DESCRIPTION
**Pull request by jmsgfhr**

### CODE REVIEW

File: manifests/xiaomi_ysl.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+    <remote name="device_xiaomi_ysl"
+        fetch="https://github.com/jmsgfhr"
+        revision="lineage-17.1" />
```
Comment (Vince1171) [inline manifests/xiaomi_ysl.xml]: this PR is targeting Halium 9
Halium 9 use Lineage 16

**Review by Vince1171 — CHANGES_REQUESTED** [review]
Hi,
thanks for your interest in the Halium project,
I've found some point that must be changed before I can merge this PR.

File: manifests/xiaomi_ysl.xml (hunk 1 above)
Comment (jmsgfhr) [reply manifests/xiaomi_ysl.xml]: Ok, I'm gonna fix this, thanks for advice.

**Review by Vince1171 — APPROVED** [review]

**Comment by Vince1171** [comment]
could you regroup all commits under one (name `add Xiaomi Redmi S2 (ysl) manifest`) and force-push?

**Comment by jmsgfhr** [comment]
Done, only one commit on this PR.

**Pull request merged**